src/table/TableWrapper.jsx

```diff
@@ -64,53 +65,76 @@ export default function TableWrapper(props) {
   }
 
   useEffect(() => {
-    const updateSize = () => setTableWidth(rootElement.clientWidth);
-    window.addEventListener('resize', updateSize);
-    tableSection.current.addEventListener('wheel', (evt) => handleScroll(evt, tableSection));
+    const resizeCallback = () => setTableWidth(rootElement.clientWidth);
+    const scrollCallback = (evt) => handleScroll(evt, tableSectionRef);
+    const focusOutCallback = (evt) => handleFocusoutEvent(evt, shouldRefocus, keyboard.blur);
+
+    window.addEventListener('resize', resizeCallback);
+    tableSectionRef.current.addEventListener('wheel', scrollCallback);
+    tableWrapperRef.current.addEventListener('focusout', focusOutCallback);
     return () => {
-      window.removeEventListener('resize', updateSize);
-      tableSection.current.removeEventListener('wheel', (evt) => handleScroll(evt, tableSection));
+      window.removeEventListener('resize', resizeCallback);
+      tableSectionRef.current.removeEventListener('wheel', scrollCallback);
+      tableWrapperRef.current.removeEventListener('focusout', focusOutCallback);
```

Comment:
Not fully understand why we are doing this, but if we want to know about resize why do we not use the useRect from Nebula?

Reply:
Not part of this PR. I think Li tried that and it didn't work, ask him 🙃

Reply:
@niekvanstaveren  do you mean the `window.addEventListener('resize', resizeCallback);`?  I gonna look into the useRect.